src/thriveopedia/fossilisation/MuseumCard.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Godot;
+
+/// <summary>
+///   Card displaying a fossilised species in the Thriveopedia museum.
+/// </summary>
+public class MuseumCard : Button
+{
+    [Export]
+    public NodePath SpeciesNameLabelPath = null!;
+
+    [Export]
+    public NodePath SpeciesPreviewPath = null!;
+
+    /// <summary>
+    ///   The fossilised species associated with this card.
+    /// </summary>
+    public Species SavedSpecies = null!;
```

Comment:
This should be a nullable property that on change would also reload the preview. Similarly to a bunch of our GUI nodes which either allow setting a value before or after attaching them to a scene.